OpenEdXMobile/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -395,7 +395,8 @@ public String getOAuthClientId() {
     }
 
     public String getOrganizationCode() {
-        return getString(ORGANIZATION_CODE, "");
+        String orgCode = getString(ORGANIZATION_CODE, "");
+        return orgCode.length() > 0 ? orgCode : null;
```

Comment:
I don't think we need to manually sanitize the config fields. We're not doing it anywhere else, even in the server responses. `null` should be the default value, and handling that properly should be enough..